modules/nw-audit-configuration.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/logging-network-security.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-policy-audit-configuration-{context}"]
+= Audit configuration
+
+The configuration for audit logging is specified as part of the OVN-Kubernetes cluster network provider configuration. The following YAML illustrates the default values for the audit logging:
+
+.Audit logging configuration
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  defaultNetwork:
+    ovnKubernetesConfig:
+      policyAuditConfig:
+        destination: "null"
+        maxFileSize: 50
+        rateLimit: 20
+        syslogFacility: local0
+----
```

Comment:
@JoeAldinger We are missing a configuration in policyAuditConfig - maxLogFiles 
It is number of log files to be retained. If the setting is 5 as soon as 6th file is created, the first will be deleted.
<pre>
policyAuditConfig:
          destination: "null"
          maxFileSize: 50
          maxLogFiles: 5
          rateLimit: 20
          syslogFacility: local0
</pre>

Reply:
This is in old documentation that is outside the scope of this PR. I can fix this in a different PR but would need to raise this as a doc's bug. @asood-rh